src/test/java/ch/aaap/harvestclient/examples/UsersApiExample.java

```diff
@@ -6,16 +6,17 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import com.typesafe.config.ConfigFactory;
+
 import ch.aaap.harvestclient.core.Harvest;
 import ch.aaap.harvestclient.domain.User;
 import ch.aaap.harvestclient.domain.param.UserCreationInfo;
-import ch.aaap.harvestclient.util.TestSetupUtil;
 
 public class UsersApiExample {
 
     private static final Logger log = LoggerFactory.getLogger(UsersApiExample.class);
 
-    private Harvest harvest = TestSetupUtil.getAdminClient();
+    private Harvest harvest = new Harvest(ConfigFactory.load());
```

Comment:
any reason for not using new Harvest() instead?

Reply:
Yes. This way the example shows that it is possible to provide your own configuration.

Reply:
Ok makes sense, but in the rest of the tests I will put in the default constructor.